server.py

```diff
@@ -119,6 +126,20 @@ def get_metrics():
     level= logging.ERROR - (args.verbose*10),
 )
 
+def consumer():
+    while True:
+        alias = alias_queue.get()
+        if alias is None:  
+            break  
+        try:
+            increment_used_column(DATABASE_FILE, alias)
+        except Exception as e:
+            logging.exception(f"Error updating used count for alias {alias}: {e}")
```

Comment:
you can do this and achieve the same result. `logging.exception` will include the exception raised on its own, its pretty cool
```suggestion
        except Exception:
            logging.exception("Error updating used count for alias {alias}")
```